JAMath.py

```diff
@@ -23,7 +22,7 @@ def __init__(self, handle):
 
         self.max_participants = 1
 
-        self.jamath_activity = Game(get_activity_root())
+        self.jamath_activity = Game(self)
```

Comment:
Would be more intuitive, with a keyword parameter
```suggestion
        self.jamath_activity = Game(activity=self)
```

Reply:
done